Livecheckables/pdftk-java.rb

```diff
@@ -0,0 +1,6 @@
+class PdftkJava
+  livecheck do
+    url :stable
```

Comment:
```suggestion
    url :head
```

The stable URL is a GitLab archive, which gets converted into the Git repo URL, so it's the same as `:head`. As we discussed, it's better to be explicit.